torch_np/_detail/implementations.py

```diff
@@ -743,6 +743,14 @@ def take_along_dim(tensor, t_indices, axis):
     return result
 
 
+def take(tensor, t_indices, axis):
+    (tensor,), axis = _util.axis_none_ravel(tensor, axis=axis)
+    axis = _util.normalize_axis_index(axis, tensor.ndim)
+    idx = (slice(None),) * axis + (t_indices, ...)
```

Comment:
TIL that the `...` literal can be used outside of square brackets. I somehow thought only the Ellipsis spelling is allowed (similar to `::-1` being only legal within `[]`). Nice!

Reply:
Yeah its a bit semantically weird to have a `...` object (aliasing `Ellipsis`), but also pretty nice for stuff like [`typing.Protocol`](https://docs.python.org/3/library/typing.html?highlight=protocol#typing.Protocol).